angularjs-portal-home/src/main/webapp/my-app/main.js

```diff
@@ -53,7 +53,7 @@ define([
         $routeProvider.
             when('/apps', marketplaceRoutes.main).
```

Comment:
More of a question for @keirserrie , what do we want to do with the old browse page?